ivy/functional/frontends/torch/spectral_ops.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+from ivy.functional.frontends.torch.func_wrapper import (
+    to_ivy_arrays_and_back,
+)
+from ivy.func_wrapper import with_supported_dtypes
+
+
+@with_supported_dtypes({"2.0.1 and below": "int"}, "torch")
+@to_ivy_arrays_and_back
+def hann_window(
+    window_length, periodic=True, *, dtype=None, requires_grad=False, out=None
+):
+    return ivy.hann_window(
+        window_length, periodic, dtype, requires_grad=requires_grad, out=out
```

Comment:
`ivy.hann_window` does not seem to support `requires_grad` kwarg. `periodic` and `dtype` are keyword-only arguments, therefore it should be kwargs

Reply:
yes, I tried yesterday but it fails